CHANGELOG.md

```diff
@@ -1,5 +1,148 @@
+
+  ```js
+  function round(value) {
+    return Math.round(value * 1e5) / 1e5;
+  }
+  ```
+
+#### Changes
+
+- <!-- 03 --> [Container] Fix maxWidth="false" resulting in incorrect css (#25869) @mnajdova
+- <!-- 49 --> [core] Improve support for extended props in theme (#25934) @vicasas
+- <!-- 21 --> [IconButton] Support custom colors and sizes (#25890) @Vikram710
+- <!-- 34 --> [Rating] Support custom sizes (#25922) @vicasas
+- <!-- 30 --> [Select] Fix classes leaking on the DOM (#25894) @siriwatknp
+- <!-- 43 --> [Stack] Fix support of spacing falsy values (#25937) @simonecervini
+- <!-- 22 --> [Table] Migrate TablePagination to emotion (#25809) @siriwatknp
+- <!-- 26 --> [Tabs] Migrate Tabs to emotion (#25824) @siriwatknp
+- <!-- 50 --> [TextField] Remove utlity class name for margin="none" (#25969) @oliviertassinari
+- <!-- 24 --> [TextField] Make the `position` prop required in InputAdornment (#25891) @m4theushw
+- <!-- 23 --> [theme] Remove fade color helper (#25895) @m4theushw
+
+### `@material-ui/lab@5.0.0-alpha.32`
+
+- <!-- 53 --> [DateTimePicker] `date` is nullable in `onChange` (#25981) @eps1lon
+- <!-- 39 --> [internal][Pickers] Remove unused TView type argument (#25936) @eps1lon
+- <!-- 48 --> [internal][Pickers] Inline some BasePickerProps usages (#25971) @eps1lon
+- <!-- 44 --> [internal][Pickers] Entangle what *Props vs All*Props means (#25938) @eps1lon
+- <!-- 19 --> [lab] Update slot components to use overridesResolver (#25906) @mnajdova
+- <!-- 40 --> [Timeline] Remove use of nth-child in favor of nth-of-type (#25915) @wellwellmissesanderson
+- <!-- 06 --> [Timeline] Migrate Timeline to emotion (#25838) @siriwatknp
+- <!-- 55 --> [TreeView] Migrate TreeItem to emotion (#25835) @siriwatknp
+
+### `@material-ui/styled-engine@5.0.0-alpha.32`
+
+- <!-- 02 --> [styled-engine] Skip variants resolver for non root slots by default (#25865) @mnajdova
+
+### Core
+
+- <!-- 54 --> [core] Fix wrong imports to '@material-ui/styles' (#25984) @mnajdova
+- <!-- 52 --> [core] Ensure props spreading works as expected (#25939) @oliviertassinari
+- <!-- 47 --> [core] Batch small changes (#25968) @oliviertassinari
+- <!-- 45 --> [core] Fix various too wide `classes` types (#25917) @eps1lon
+- <!-- 35 --> [core] Enable trailing comma in TypeScript files (#25931) @eps1lon
+- <!-- 33 --> [core] Remove @typescript-to-proptypes-generate handlers (#25909) @eps1lon
+- <!-- 18 --> [core] Update slots components to enable flatten specificity for overrides (#25853, #25864, #25881, #25884, #25887, #25904, #25892) @mnajdova
+- <!-- 05 --> [Drawer] Fix classes forwarded to DOM node for docked drawer (#25870) @mnajdova
+- <!-- 12 --> [system] Add missing `main` entry for styleFunctionSx (#25885) @eps1lon
+- <!-- 27 --> [test] Add current behavior of inverleaving elements on mousedown (#25903) @eps1lon
+- <!-- 20 --> [test] Add test validator to improve DX (#25854) @siriwatknp
+
+### Docs
```

Comment:
So far, we have made docs come before core before more important

Reply:
Moved the Docs before the Core